src/Raven.Server/Documents/Handlers/Processors/Subscriptions/AbstractSubscriptionsHandlerProcessorForGetSubscription.cs

```diff
@@ -47,7 +47,7 @@ await using (var writer = new AsyncBlittableJsonTextWriterForDebug(context, Serv
             }
         }
 
-        internal static void WriteGetAllResult(AsyncBlittableJsonTextWriter writer, IEnumerable<SubscriptionState> subscriptions, ClusterOperationContext context)
+        internal static void WriteGetAllResult(AsyncBlittableJsonTextWriterForDebug writer, IEnumerable<SubscriptionState> subscriptions, ClusterOperationContext context)
```

Comment:
This shouldn't be generic as well?

Reply:
No need, no. The only caller is for debugging purposes.